scripts/bluepill.sh

```diff
@@ -85,7 +85,6 @@ bluepill_build_sample_app()
     -workspace Bluepill.xcworkspace \
     -scheme BPSampleApp \
     -sdk iphonesimulator \
-    -destination 'platform=iOS Simulator,name=iPhone 7,OS=11.1' \
```

Comment:
why is this deleted ?

Reply:
It's unnecessary. When building the app, if the destination doesn't exit (for example because you upgraded Xcode and there is no 11.1 iOS runtime) the build fails.

Reply:
You mean exist instead of exit ? :)

Reply:
lol yeah...